apps/andi/endpoint.ex

```diff
@@ -1,9 +0,0 @@
-defmodule Andi.Endpoint do
-  use Phoenix.Endpoint, otp_app: :andi
-
-  plug Plug.Static,
-    at: "/",
-    from: :andi,
-    gzip: false,
-    only: ~w(css fonts images js favicon.ico)
-end
```

Comment:
Oh so do we not need this since we're now hosting the font ourself?

Reply:
No, we do need it. But it already exists under apps/andi/lib/andi_web/endpoint.ex.